src/pages/movieList.js

```diff
@@ -2,16 +2,25 @@
 
 import {createMovieCard} from '../components/card.js';
 import {renderPageLayout} from '../layouts/createLayout.js';
+import {getTopRatedFilms} from '../services/api.js';
 
 export function createMovieList() {
     const arrayWithCards = [];
 
-    for (let i = 0; i < 20; i++) {
-        arrayWithCards.push(createMovieCard());
-    }
+    getTopRatedFilms().then(result => {
 
-    renderPageLayout(arrayWithCards);
+        for (let i = 0; i < result.length; i++) {
```

Comment:
pls, use the here one of array's methods

Reply:
fixed